src/modules/75-cd/components/PipelineSteps/CommandScripts/Commands/ScriptCommandEdit.tsx

```diff
@@ -66,7 +74,26 @@ export function ScriptCommandEdit(props: ScriptCommandEditProps): React.ReactEle
           }}
         />
       </Container>
+      <FormInput.RadioGroup
+        name="spec.source.type"
+        label={getString('cd.steps.commands.selectScriptLocation')}
+        items={[
+          {
+            label: getString('cd.steps.commands.locationFileStore'),
+            value: 'Harness'
```

Comment:
`LocationType` enum could be used here
```suggestion
            value: LocationType.HARNESS
```